app/src/gui/components/notebook/index.tsx

```diff
@@ -233,100 +276,120 @@ export default function NotebookComponent(props: NotebookComponentProps) {
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'last_updated'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                    </TableBody>
-                  </Table>
-                </TableContainer>
-              </Grid>
-              <Grid item xs={12} sm={12} md={12} lg={4}>
-                <RangeHeader
-                  project={project}
-                  handleAIEdit={handleNotebookTabChange}
+                  Edit
+                </Typography>
+              </IconButton>
+            </Box>
+
+            <Box sx={{p: 2}}>
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Name:</strong>{' '}
+                <MetadataRenderer
+                  project_id={project.project_id}
+                  metadata_key={'name'}
+                  chips={false}
                 />
-              </Grid>
-            </Grid>
+              </Typography>
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Description:</strong>{' '}
+                <MetadataRenderer
+                  project_id={project.project_id}
+                  metadata_key={'pre_description'}
+                  chips={false}
+                />
+              </Typography>
+
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Status:</strong>{' '}
+                <MetadataRenderer
+                  project_id={project.project_id}
+                  metadata_key={'project_status'}
+                  chips={true}
+                />
+              </Typography>
+
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Lead Institution:</strong>{' '}
+                <MetadataRenderer
+                  project_id={project.project_id}
+                  metadata_key={'lead_institution'}
+                  chips={false}
+                />
+              </Typography>
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px', textAlign: 'left'}}
+              >
+                <strong>Project Lead:</strong>{' '}
+                <MetadataRenderer
+                  project_id={project.project_id}
+                  metadata_key={'project_lead'}
+                  chips={false}
+                />
+              </Typography>
+
+              <ArchiveSurvey
```

Comment:
Please don't include links or buttons that don't do anything. Until we know what archiving means we can't implement it and having a button in the UI that does nothing is confusing for users.  Remember, Fieldmark has users right now even though BSS won't have users until next year.

Reply:
Hey steve, 
Sure, that makes sense.
I think I'll just remove the archive section as a whole for now.